src/test/cobol/MOCKTEST/MockCallTest.cut

```diff
@@ -138,4 +138,49 @@
                 MOVE "Local PROG1" TO VALUE-1
            END-MOCK
            PERFORM 600-MAKE-CALL
-           EXPECT VALUE-1 TO BE "Global PROG1"
\ No newline at end of file
+           EXPECT VALUE-1 TO BE "Global PROG1"
+
+           TestCase "Mock of call with exception handling works"
+           MOCK CALL 'PROGRAM' USING 
+             VALUE-1
+                MOVE "worked" TO VALUE-1
+           END-MOCK
+           MOVE 'TEST' TO VALUE-1
+           PERFORM 900-MAKE-CALL
+           EXPECT VALUE-1 TO BE "worked"
+
+           TestCase "Mock of call with exception handling ending with period works"
+           MOCK CALL 'PROGRAM' USING VALUE-1
+                MOVE "worked" TO VALUE-1
+           END-MOCK
+           MOVE 'TEST' TO VALUE-1
+           PERFORM 1000-MAKE-CALL
+           EXPECT VALUE-1 TO BE "worked"
+
+           TestCase "Mock of call with a new call as exception handling works"
+           MOCK CALL 'PROGRAM' USING VALUE-1
+                MOVE "worked" TO VALUE-1
+           END-MOCK
+           MOVE 'TEST' TO VALUE-1
+           PERFORM 1100-MAKE-CALL
+           EXPECT VALUE-1 TO BE "worked"
+
+           TestCase "Mock of call with a new call as exception handling ending with period works"
+           MOCK CALL 'PROGRAM' USING VALUE-1
+                MOVE "worked" TO VALUE-1
+           END-MOCK
+           MOCK CALL 'PROGRAM2' USING VALUE-1
+                MOVE "worked" TO VALUE-1
```

Comment:
Please change the value being moved to value-1.
When using "worked" in both mocks, we cannot see, which of the mocks where executed.